tests/modules/suite2p_registration/test_suite2p_registration_suite2p_utils.py

```diff
@@ -147,3 +149,138 @@ def test_compute_reference(self):
+
+        with patch('ophys_etl.modules.suite2p_registration.suite2p_utils.'
+                   'compute_reference', Mock), \
+             patch('ophys_etl.modules.suite2p_registration.suite2p_utils.'
+                   'create_ave_image', create_ave_image_mock):
+            best_result = optimize_motion_parameters(
+                initial_frames=self.frames,
+                suite2p_args=suite2p_args,
+                logger=print)
+        self.assertAlmostEqual(best_result['acutance'], 1.0)
+        self.assertAlmostEqual(best_result['smooth_sigma'], 1.15)
+        self.assertAlmostEqual(best_result['smooth_sigma_time'], 1.0)
+
+    def test_create_ave_image_mock(self):
+        """Run test data through to assert that the average image is
+        created as expected.
+        """
+        suite2p_args = {'h5py': self.h5_file_loc,
+                        'h5py_key': 'input_frames',
+                        'smooth_sigma': 1.15,
+                        'smooth_sigma_time': 1.0}
+        add_required_parameters(suite2p_args)
+        mock_frames = np.ones((self.n_frames,
+                               self.n_frames,
+                               self.n_frames),
+                              dtype=int)
+        max_shfit = 10
+
+        def register_frames_mock(refAndMasks, frames, ops):
+            return (mock_frames,
+                    self.rng.integers(-max_shfit,
+                                      max_shfit,
+                                      size=self.n_frames),
+                    self.rng.integers(-max_shfit,
+                                      max_shfit,
+                                      size=self.n_frames),
+                    np.arange(self.n_frames),
+                    np.arange(self.n_frames),
+                    np.arange(self.n_frames),
+                    np.arange(self.n_frames))
+
+        with patch('ophys_etl.modules.suite2p_registration.suite2p_utils.'
+                   'register_frames', register_frames_mock):
+            result = create_ave_image(self.original_reference, suite2p_args)
+        self.assertTrue(np.allclose(result['ave_image'],
+                                    np.ones((self.n_frames,
+                                             self.n_frames))))
+        self.assertEqual(result['dy_max'], 10)
+        self.assertEqual(result['dx_max'], 10)
+
+    def test_create_ave_image(self):
+        """Run test data through to assert that the average image is
+        created as expected.
+        """
+        suite2p_args = {'h5py': self.h5_file_loc,
+                        'h5py_key': 'input_frames',
+                        'maxregshift': 0.2,
+                        'smooth_sigma': 1.15,
+                        'smooth_sigma_time': 1.0}
+        add_required_parameters(suite2p_args)
+        result = create_ave_image(self.original_reference,
+                                  suite2p_args)
+        self.assertTrue(np.allclose(result['ave_image'],
```

Comment:
`np.testing.assert_allclose` will probably give more data in the event of a failure. Any reason not to use that?

Reply:
No reason. Mostly just wasn't that familiar with the numpy.testing module.